### PR TITLE
mParticle integration publishing

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/faqs/yes-you-can-faq**](https://docs-4nhirmvh4.avo-dev.app/faqs/yes-you-can-faq) | [**/workspace/integrations**](https://docs-4nhirmvh4.avo-dev.app/workspace/integrations) |
|:-:|:-:|
|<a href="https://docs-4nhirmvh4.avo-dev.app/faqs/yes-you-can-faq">Screenshot of /faqs/yes-you-can-faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/0AmW2sGzU9jdr3CZeWg2UJOhNWfUY1bFl0WuDgLGRvG1-pxA-ilB8eCEIfqZx2DvA5hOT_eUarguRRrwZT7Q4Q.png">(view full size)</a> |<a href="https://docs-4nhirmvh4.avo-dev.app/workspace/integrations">Screenshot of /workspace/integrations</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/xQ4WUYBaV42oF8qWqwaYAeBzM2CmNa-uuQkde7Hh-vDnWK7IhK-XqcHdTIHTmwA5iUDqdY1l8dDNS9Hrm9AiIA.png">(view full size)</a> |



| [**…acking-plan/publishing**](https://docs-4nhirmvh4.avo-dev.app/workspace/tracking-plan/publishing) |
|:-:|
|<a href="https://docs-4nhirmvh4.avo-dev.app/workspace/tracking-plan/publishing">Screenshot of /workspace/tracking-plan/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-1Tmk3RolYBX_XO5idBuqHDZ8aowg_aR3_aFjmImvYhYbHlmmIbEf-P9l4vandhiPVLstfLASKM9_qPnlYieqg.png">(view full size)</a> |







Commit d28687c2dd168ba97569ba0fea72f4a013ccea07 (https://docs-4nhirmvh4.avo-dev.app).

File: pages/workspace/integrations.mdx [hunk 1]
```diff
@@ -46,6 +47,8 @@ With Avo you can _<Link passHref href="/workspace/tracking-plan/publishing">Publ
 ### Amplitude Govern
 <Link passHref href="/workspace/tracking-plan/publishing#amplitude-govern">Publish to Amplitude Govern</Link> to sync your analytics schema to downstream
 
+### mParticle Data Master
+<Link passHref href="/workspace/tracking-plan/publishing#mparticle">Publish to Amplitude Govern</Link> to sync your analytics schema to downstream
```
Comment (hugihlynsson) [inline pages/workspace/integrations.mdx:51]:
```suggestion
<Link passHref href="/workspace/tracking-plan/publishing#mparticle">Publish to mParticle Data Master</Link> to sync your analytics schema to downstream
```

**Review by hugihlynsson — CHANGES_REQUESTED** [review]

File: pages/workspace/tracking-plan/publishing.mdx [hunk 2]
```diff
@@ -175,6 +176,33 @@ Avo will override all events in the RudderStack Tracking Plan that are being imp
...
+- **mParticle Client ID**: Your client ID issued by mParticle
+- **mParticle Client Secret**: Your client secret issued by mParticle
+- **mParticle Workspace ID**: The ID of the workspace containing your data plans
+- **mParticle Plan ID**: The ID of the Data Plan to update
+
+### Publishing to mParticle
+
+Avo will create a new version for each event that you publish.
+
+Avo will override all events in the mParticle Tracking Plan that are being imported from Avo. Any changes made to an event in mParticle Tracking Plan between Avo publish will be overridden on next Avo publish.
```
Comment (bragur) [inline pages/workspace/tracking-plan/publishing.mdx:202]:
```suggestion
Avo will override all events in the mParticle Tracking Plan that are being imported from Avo. Any changes made to an event in an mParticle Tracking Plan between publishing from Avo will be overridden on the next Avo publish.
```

I was a bit confused here, does this make sense?

File: pages/workspace/integrations.mdx [hunk 3]
```diff
@@ -46,6 +47,8 @@ With Avo you can _<Link passHref href="/workspace/tracking-plan/publishing">Publ
 ### Amplitude Govern
 <Link passHref href="/workspace/tracking-plan/publishing#amplitude-govern">Publish to Amplitude Govern</Link> to sync your analytics schema to downstream
 
+### mParticle Data Master
+<Link passHref href="/workspace/tracking-plan/publishing#mparticle">Publish to mParticle Data Master</Link> to sync your analytics schema to downstream
```
Comment (bragur) [inline pages/workspace/integrations.mdx:51]:
```suggestion
<Link passHref href="/workspace/tracking-plan/publishing#mparticle">Publish to mParticle Data Master</Link> to sync your analytics schema downstream
```

I think 🤔

**Review by bragur — APPROVED** [review]
Looks good! Just a couple of comments with possible improvements.

File: pages/faqs/yes-you-can-faq.mdx [hunk 4]
```diff
@@ -97,6 +97,10 @@ Yes, you can! We integrate directly with Segment Protocols. <Link passHref href=
 
 Yes, you can! We integrate directly with RudderStack. <Link passHref href="/workspace/tracking-plan/publishing#rudderstack"><a>Find out more here</a></Link>.
 
+### Can I publish my Avo tracking plan to mParticle?
+
+Yes, you can! We integrate directly with mParticle. <Link passHref href="/workspace/tracking-plan/publishing#mparticle"><a>Find out more here</a></Link>.
```
Comment (logason) [inline pages/faqs/yes-you-can-faq.mdx:102]:
```suggestion
Yes, you can! We integrate directly with mParticle Data Master. <Link passHref href="/workspace/tracking-plan/publishing#mparticle"><a>Find out more here</a></Link>.
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 5]
```diff
@@ -175,6 +176,33 @@ Avo will override all events in the RudderStack Tracking Plan that are being imp
...
 
+<img
+  src={require('../../../images/publishing_mparticle.svg')}
+  alt="mParticle"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="mparticle"></a> mParticle Data Master
+
+The [mParticle integration](https://docs.mparticle.com/guides/data-master/introduction/) enables you to publish your Avo Tracking Plan into mParticle. You can configure the integration to publish whenever a branch is merged or with a push of a button.
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:187]: I assumed the link would take me to some detailed material about the Avo<>mParticle integration, I suggest this change to better clarify where the links take you:
```suggestion
The [mParticle Data Master](https://docs.mparticle.com/guides/data-master/introduction/) integration enables you to publish your Avo Tracking Plan into mParticle Data Master. You can configure the integration to publish whenever a branch is merged or with a push of a button.
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 6]
```diff
@@ -175,6 +176,33 @@ Avo will override all events in the RudderStack Tracking Plan that are being imp
...
+
+### Configure the mParticle integration
+
+To enable the direct integration with mParticle Data Master, you need to configure it with the following parameters from mParticle.
+
+- **mParticle Client ID**: Your client ID issued by mParticle
+- **mParticle Client Secret**: Your client secret issued by mParticle
+- **mParticle Workspace ID**: The ID of the workspace containing your data plans
+- **mParticle Plan ID**: The ID of the Data Plan to update
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:196]: Can we add more details here on how you find these things in the mParticle UI? I had a hard time finding the workspace id at first, and the client id/secret also requires a bit of work.

**Review by logason — COMMENTED** [review]
Looking good! Here are few suggestions:

File: pages/workspace/tracking-plan/publishing.mdx (hunk 6 above)
Comment (bjornj12) [reply pages/workspace/tracking-plan/publishing.mdx:196]: even a hyperlink to the location would be nice if that is possible

**Review by bjornj12 — APPROVED** [review]
LGTM when comments have been addressed

**Review by hugihlynsson — APPROVED** [review]

File: pages/faqs/yes-you-can-faq.mdx (hunk 4 above)
Comment (alex-tpom6oh) [reply pages/faqs/yes-you-can-faq.mdx]: I feel like they might change this name at some point

**Pull request merged**